cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/timeseries/FactTable.java

```diff
@@ -143,12 +144,23 @@ public FactScanner scan(FactScan scan) throws Exception {
   /**
    * Delete entries in fact table.
    * @param scan
+   * @param measurePrefixMatch - we have this to support V2 API for deleting data whose
```

Comment:
v2 API has option to delete metrics with a metrics Prefix. in order to support that we need to scan and delete the rows which match the given metrics prefix, but if the entire metric name is known we can directly do deleteRange on that without scan. if we have to remove this prefixMatch, then ill make the scan and delete as default.